lib/Base/Image.cpp

```diff
@@ -459,18 +483,42 @@ void glow::loadImagesAndPreprocess(const llvm::ArrayRef<std::string> &filenames,
   case ImageLayout::NHWC:
     batchDims = {numImages, imgHeight, imgWidth, numChannels};
     break;
+  default:
+    LOG(FATAL) << "Unexpected layout\n";
```

Comment:
nit: Shouldn't be necessary, since we already covered the only 2 enum values in the `ImageLayout` enum. Were you seeing issues here?